src/sage/databases/oeis.py

```diff
@@ -890,7 +891,7 @@ def old_IDs(self):
             A000045: Fibonacci numbers: F(n) = F(n-1) + F(n-2) with F(0) = 0 and F(1) = 1.
 
             sage: f.old_IDs()                           # optional -- internet
-            ('M0692', 'N0256')
+            ('M0692', 'N0256'...)
```

Comment:
I don't think that this is the correct fix.   Rather, we should probably first strip of the date.  For example, `f._field("I")` currently gives `['M0692 N0256 #2128 Dec 18 2023 00:21:04']`.  Unfortunately, this is not documented at https://oeis.org/eishelp1.html.  Maybe we should ask?  In any case, it seems quite clear that everything after the `#` should be ignored for our purposes.

Reply:
You are right, we should ask.

Reply:
I sent email to seqfan.  I suspect that the number after the #-sign is the version number.  In any case, we should strip it off, it appears that it is always there.

Reply:
Should be ok with [ed94024](https://github.com/sagemath/sage/pull/36976/commits/ed9402464d834ad8f8b5f1fd6aff7db8bb814093).